public/index.html

```diff
@@ -26,20 +26,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Upswyng</title>
+    <meta name="description" content="UpSwyng is a mobile-ready, digital directory of resources to assist the homeless and at-risk in Boulder County, Colorado."/>
```

Comment:
people experiencing homelessness?

Reply:
good call. This was run by Mike and [updated](https://github.com/CodeForBoulder/upswyng/pull/89/commits/e75ff3e4baf2e606994aad3be523acabca753b5d)